packages/react-query-devtools/src/devtools.tsx

```diff
@@ -681,6 +686,48 @@ export const ReactQueryDevtoolsPanel = React.forwardRef<
             queryClient={queryClient}
           />
         ) : null}
+        {isOpen && (showCloseButton || position) ? (
+          <div
+            style={{
+              position: 'absolute',
+              zIndex: 99999,
+              margin: '.5em',
+              bottom: 0,
+              left: 0,
+              display: 'flex',
+              placeItems: 'center',
+            }}
+          >
+            {showCloseButton ? (
+              <Button
+                type="button"
+                aria-controls="ReactQueryDevtoolsPanel"
+                aria-haspopup="true"
+                aria-expanded="true"
+                {...(otherCloseButtonProps as Record<string, unknown>)}
+                onClick={(e) => {
+                  setIsOpen(false)
+                  onCloseClick?.(e)
+                }}
+              >
+                Close
+              </Button>
+            ) : null}
+            {position && onPositionChange ? (
+              <Select
```

Comment:
I think we'd also need a way to hide this select. One reason to hide the closeButton was because it overlaps with queries if you have many of them, and the Select in this position probably has the same drawback.

Reply:
should I move the position select on the top or just add a flag to hide it

Reply:
is there enough place on the top? But yeah, I would like to avoid more flags ...

Reply:
I will try to squeeze it in there